### PR TITLE
Unit tests for vppagent/backend.go::BuildVppIfName(), ProcessEndpoint and ProcessClient() functions

### PR DESCRIPTION
**Pull request by amberimam**

### CODE REVIEW

File: pkg/universal-cnf/vppagent/backend_test.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	"os"
+	"testing"
+	"github.com/stretchr/testify/assert"
+
+	"go.ligato.io/vpp-agent/v3/proto/ligato/vpp"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	connectioncontext "github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
+	"github.com/networkservicemesh/networkservicemesh/sdk/common"
+	interfaces "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/interfaces"
+	vpp_l3 "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3"
```
Comment (ana-moga) [inline pkg/universal-cnf/vppagent/backend_test.go]: sort imports

File: pkg/universal-cnf/vppagent/backend_test.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		srcIpAddr = "192.168.22.1"
+		srcIpRoute = "192.168.0.1/16"
+		workspaceEnv = "workspace"
+	)
+
+	podName := "helloworld-ucnf-78956df47b-r5fj4"
+	b := UniversalCNFVPPAgentBackend{}
+	vppconfig := &vpp.ConfigData{}
+	serviceName := "ucnf"
+	ifName := "endpoint0"
```
Comment (ana-moga) [inline pkg/universal-cnf/vppagent/backend_test.go]: I would suggest that podName, workspaceEnv, serviceName, ifName/defaultIfName, and other variables to be set as a const () in this file, after imports. I see they are used in multiple places. And srcIpAddr and srcIpRoute into scrIpAddrEndpoint/Client maybe

File: pkg/universal-cnf/vppagent/backend_test.go (hunk 1 above)
Comment (amberimam) [reply pkg/universal-cnf/vppagent/backend_test.go]: Sure, will do!

File: pkg/universal-cnf/vppagent/backend_test.go (hunk 2 above)
Comment (amberimam) [reply pkg/universal-cnf/vppagent/backend_test.go]: Sure, will do! Thanks for the review.

File: pkg/universal-cnf/vppagent/backend_test.go [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+	expected := podName
+	got := b.buildVppIfName("", "", conn)
+
+	assert.Equal(t, expected, got)
+}
+
+func TestBuildVppIfNameDefault(t *testing.T) {
+
+	serviceName := "ucnf"
```
Comment (CosminNechifor) [inline pkg/universal-cnf/vppagent/backend_test.go:62]: I guess you could use the const that you've added above at line 24.

**Review by CosminNechifor — APPROVED** [review]
LGTM.

**Review by ana-moga — APPROVED** [review]

File: pkg/universal-cnf/vppagent/backend_test.go (hunk 3 above)
Comment (amberimam) [reply pkg/universal-cnf/vppagent/backend_test.go:62]: Ahh good catch, I will fix it in my next PR! :)

**Pull request merged**